perun-web-gui/src/main/java/cz/metacentrum/perun/webgui/model/Attribute.java

```diff
@@ -209,7 +209,7 @@ public final native boolean setValue(String newValue) /*-{
 				return false;
 			}
 		}
-		if (this.type == "java.util.ArrayList") {
+		if (this.type == "java.util.ArrayList" || this.type == "java.util.LargeArrayList") {
```

Comment:
@stavamichal This is only usage in plain javascript, references in java are using constant.